CMakeLists.txt

```diff
@@ -74,7 +75,8 @@ find_package(PCL REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system thread date_time python)
 
 find_package(APR REQUIRED)
-find_package(UIMA REQUIRED)
+find_package(MongoClientLibrary REQUIRED)
+find_package(uima REQUIRED)
```

Comment:
please call the ros package uimacpp_ros just to avoid any confusion and find_package it with the rest of the ROS packages in line 54